### PR TITLE
Added basic readme description

### LINKED ISSUE #7: fill in readme
it's currently blank!

### PR DESCRIPTION
**Pull request by TanzHasan**
Created a barebones README with instructions on how to run this tool
Closes #7

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,20 @@
...
+
+Run the following scripts on the first run. If you want to test for different compile_commands.json you will need to rerun this:
+
+```bash
+python process_ast.py -c compile_commands.json > ast_dump.txt
+python ast_json.py -f ./ast_dump.txt
+```
+
+Replace compile_commands.json with the relative or absolute location. 
+Optionally add -c and a filename to the ast_json invocation if you want to specify a file other than the auto generated outfile.json.
```
Comment (nickdesaulniers) [inline README.md:12]: so ast_json.py produces outfile.json that's consumed by hierarchical_agglomeration.py? Can you make that more explicit?

I think it should be possible at some point to merge process_ast.py and ast_json.py and just produce outfile.json (or have hierarchical_agglomeration drive the whole thing.

**Review by nickdesaulniers — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (TanzHasan) [reply README.md:12]: Yeah okay I will make that more explicit. I am planning to merge the two, that is an issue that I will raise for the future.

Comment (nickdesaulniers) [reply README.md:12]: file an issue to track that, the close it via pr

Comment (TanzHasan) [reply README.md:12]: Yep, just filed.

**Pull request merged**